src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeFixes/FixAllContextExtensions.cs

```diff
@@ -2,18 +2,18 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.Linq;
 using Microsoft.CodeAnalysis.CodeFixesAndRefactorings;
-using Microsoft.CodeAnalysis.Shared.Utilities;
 
 namespace Microsoft.CodeAnalysis.CodeFixes
 {
     internal static class FixAllContextExtensions
     {
-        public static IProgressTracker GetProgressTracker(this FixAllContext context)
+        public static IProgress<CodeAnalysisProgress> GetProgressTracker(this FixAllContext context)
         {
 #if CODE_STYLE
-            return NoOpProgressTracker.Instance;
+            return CodeAnalysisProgress.Null;
 #else
             return context.ProgressTracker;
 #endif
```

Comment:
if we can make CodeAnalysisProgress public, then this method goes away entirely.  Anyone will be able to access the progress object off of FixAllContext.